src/config/ConfigNames.ts

```diff
@@ -179,12 +179,21 @@ class ConfigNames {
     public static readonly THUNDRA_TRACE_INTEGRATIONS_AWS_SES_MAIL_DESTINATION_MASK: string =
         'thundra.agent.trace.integrations.aws.ses.mail.destination.mask';
 
+    public static readonly THUNDRA_TRACE_INTEGRATIONS_GOOGLE_PUBLISH_BODY_MASK: string =
+        'thundra.agent.trace.integrations.google.publish.body.mask';
+
+    public static readonly THUNDRA_TRACE_INTEGRATIONS_GOOGLE_SUBSCRIPTION_BODY_MASK: string =
+        'thundra.agent.trace.integrations.google.subscription.body.mask';
+
     public static readonly THUNDRA_TRACE_INTEGRATIONS_HAPI_DISABLE: string =
         'thundra.agent.trace.integrations.hapi.disable';
 
     public static readonly THUNDRA_TRACE_INTEGRATIONS_KOA_DISABLE: string =
         'thundra.agent.trace.integrations.koa.disable';
 
+    public static readonly THUNDRA_TRACE_INTEGRATIONS_GOOGLE_SUBSCRIPTION_DISABLE: string =
```

Comment:
`THUNDRA_TRACE_INTEGRATIONS_GOOGLE_PUBSUB_DISABLE` to disable both tracing publish and subscribe operations